assets/prototype/domain/shared/data/queries/entities/useUpdateDatetimeList.ts

```diff
@@ -0,0 +1,17 @@
+import { DatetimesList } from '../../../../eventEditor/data/types';
+import { WriteQueryOptions } from '../../../../eventEditor/data/queries/types';
+import useUpdateEntityList from './useUpdateEntityList';
+import useDatetimeQueryOptions from '../../../../eventEditor/data/queries/datetimes/useDatetimeQueryOptions';
+import { CacheUpdaterFn } from '../types';
```

Comment:
this shared logic seems to be pulling in a lot of stuff from the `eventEditor` domain...
can that other logic be shared too? or is this logic maybe still too coupled to the event editor?
maybe this shouldn't be shared if it is tied to the event editor this much?

Reply:
Yes, didn't notice that. Thank you :)